roamance/src/components/social/post/post-card.tsx

```
@@ -0,0 +1,787 @@
+                objectFit="cover"
+                className="transition-transform duration-500 ease-out group-hover:scale-105"
+              />
+            ) : (
+              <video
+                src={
+                  post.video_paths[currentImageIndex - post.image_paths.length]
+                }
+                className="w-full h-full object-cover"
+                controls
+                preload="metadata"
+              />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)